myfile.txt

```diff
@@ -0,0 +1,1904 @@
+.docker/Dockerfile.devenv
```

Comment:
Was this file meant to be included?

Reply:
No, myfile.txt is just a remanent of the method I used to grab the commit dates for each file locally. This was implemented hastily to experience how it will look and feel. There's no intent to use/merge any of this. The implementation will need to be better designed if we decide to do it.